test/evergreen.yml

```diff
@@ -5334,3 +5334,43 @@ buildvariants:
     - name: unit-test-nonstandalone
     - name: unit-test-long-nonstandalone
       distros: ubuntu2004-arm64-large
+
+- name: amazon2-arm64
+  display_name: "~ Amazon Linux 2 ARM64"
```

Comment:
The leading `~` usually indicates a builder is either the least important or most expensive - neither applies to this builder?